### PR TITLE
Update cri-tools to v1.19.0

### PR DESCRIPTION
**Pull request by xmudrii**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:

This updates cri-tools to the latest release ([v1.19.0](https://github.com/kubernetes-sigs/cri-tools/releases/tag/v1.19.0)).

**Does this PR introduce a user-facing change?**:

```release-note
Update cri-tools to [v1.19.0](https://github.com/kubernetes-sigs/cri-tools/releases/tag/v1.19.0)
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs
None
```

/assign @saschagrunert @cpanato 
cc @kubernetes/release-engineering

### CODE REVIEW

**Review by cpanato — APPROVED** [review]
/lgtm

**Review by saschagrunert — APPROVED** [review]
/lgtm
/approve

**Review by hasheddan — APPROVED** [review]
/lgtm

**Comment by saschagrunert** [comment]
/assign @mikedanese 

for approval

**Comment by justaugustus** [comment]
Great job, Marko!
/lgtm
/approve
/milestone v1.20

File: build/workspace.bzl [hunk 1]
```diff
@@ -68,7 +68,7 @@ def cri_tarballs():
             name = "cri_tools_%s" % arch,
             downloaded_file_path = "cri_tools.tgz",
             sha256 = sha,
-            urls = mirror("https://github.com/kubernetes-incubator/cri-tools/releases/download/v%s/crictl-v%s-%s.tar.gz" % (CRI_TOOLS_VERSION, CRI_TOOLS_VERSION, arch)),
+            urls = mirror("https://github.com/kubernetes-sigs/cri-tools/releases/download/v%s/crictl-v%s-%s.tar.gz" % (CRI_TOOLS_VERSION, CRI_TOOLS_VERSION, arch)),
```
Comment (justaugustus) [inline build/workspace.bzl:71]: @xmudrii -- One stylistic nit, which does not need to be addressed, it's just a tip for future PRs...

When you change artifact locations, try to do so as a separate commit; it makes it easier to track down, edit, or remove (if you need to later).

Comment (xmudrii) [reply build/workspace.bzl:71]: @justaugustus Thank you for the tip!

**Pull request merged**